app/src/main/java/com/github/fribourgsdp/radio/data/view/PlaylistsFragmentHolderActivity.kt

```diff
@@ -1,5 +1,6 @@
 package com.github.fribourgsdp.radio.data.view
 
+import android.content.Intent
```

Comment:
A minor change but I think you can remove this import.

Reply:
fixed in latest commit